addons/easy_charts/utilities/containers/canvas/plot_box/plot_box.gd

```diff
@@ -1,56 +1,32 @@
 extends Control
 class_name PlotBox
 
-signal function_point_entered(point, function)
-signal function_point_exited(point, function)
-@onready var tooltip: DataTooltip = $Tooltip
+# TODO: These signals have been removed. If anyone needs them, we can bring
+# them back, but from the Chart, not from the PlotBox.
+#signal function_point_entered(point, function)
+#signal function_point_exited(point, function)
 
 var focused_point: Point
 var focused_function: Function
 
-var x_labels_function: Callable = Callable()
-var y_labels_function: Callable = Callable()
-
 var box_margins: Vector2 # Margins relative to this rect, in order to make space for ticks and tick_labels
 var plot_inner_offset: Vector2 = Vector2(15, 15) # How many pixels from the broders should the plot be
 
+# TODO: Remove
 var chart_properties: ChartProperties
```

Comment:
Explanation: This is still needed by function plotters. In a follow-up PR, this will be replaced by function plotters having direct access to the chart.